test/e2e/components/vl-multi-select.js

```diff
@@ -1,6 +1,136 @@
-const { VlElement } = require('vl-ui-core').Test;
+const { VlSelect } = require('vl-ui-select').Test;
+const { Pill } = require('./pill');
+const { By } = require('selenium-webdriver');
+
+class VlMultiSelect extends VlSelect {
+    async _getRoot() {
+        return this.findElement(By.xpath('../..'));
+    }
+
+    async _getItemList() {
+        const root = await this._getRoot();
+        return root.findElement(By.css('.vl-select__list > div'));
+    }
+
+    async _getItemsToSelect() {
+        const selectList = await this._getItemList();
+        return selectList.findElements(By.css('.vl-select__item'));
+    }
+
+    async _getPills() {
+        const root = await this._getRoot();
+        const pills = await root.findElements(By.css('.vl-pill'));
+        return Promise.all(pills.map(pill => new Pill(this.driver, pill)));
+    }
+
+    async _getPillByValue(value) {
+        const pills = await this._getPills();
+        const elements = await Promise.all(pills.map(async (pill) => {
+            const value = await pill.getValue();
+            return { webElement: pill, value: value };
+        }));
+        return elements.filter(e => e.value == value)[0].webElement;
+    }
+
+    async _getSelectedOptions() {
+        return this.findElements(By.css('option'));
+    }
```

Comment:
naar de options kijken is denk ik niet nodig want je kan ook alles uit de pills halen